newIDE/app/src/Version/index.js

```diff
@@ -13,3 +14,22 @@ export const getGDCoreVersion = (): string => {
   gdCoreVersionString = gd ? gd.VersionWrapper.fullString() : 'Unknown';
   return gdCoreVersionString;
 };
+
+export const getPlatform = (): string => {
+  let platformString;
+  if (process.platform === 'darwin') {
+    platformString = 'macOS';
+  } else {
+    platformString = process.platform;
+  }
+
+  return platformString;
+};
+
+export const getSystemVersion = (): string => {
+  return process.getSystemVersion
+};
+
+export const getArch = (): string => {
+  return process.arch;
+};
```

Comment:
Might be a good idea to also add:
* https://developer.mozilla.org/en-US/docs/Web/API/NavigatorID/userAgent
* A version of getPlatform to get the system in a webapp: `https://developer.mozilla.org/en-US/docs/Web/API/Navigator/oscpu`
* Maybe we should put getSystemVersion inside getPlatform. Let's in this case rename "getPlatform" to "getOperatingSystemDescriptiveName"